module_utils/network_lsr/argument_validator.py

```diff
@@ -413,6 +414,73 @@ def _validate_post(self, value, name, result):
 
         return result
 
+class ArgValidatorIPRule(ArgValidatorDict):
+    def __init__(self, name, family=None, required=False, default_value=None):
+        ArgValidatorDict.__init__(
+            self,
+            name,
+            required,
+            nested=[
+                ArgValidatorIP(
+                    "from", family=family, required=False, plain_address=False
+                ),
+                ArgValidatorNum("from_prefix", default_value=32, val_min=0, val_max=32),
```

Comment:
I think `from_prefix` and `from` should be merged as `ArgValidatorIPAddr` types. Likewise for `to`.

Reply:
I don't disagree; but in routes, network and prefix are split as well, so I did not want to introduce any confusions.